### ISSUE TITLE
Backend url

### ISSUE THREAD

**Issue by medrockstar** [STS-Rosario/carpoolear_backend#99]
Hi,

what's the url of backend ?

**Comment by p4bl1t0** [comment]
The backend url is what you define. You have to hosted and publish on Apache.

Or also you could try with this project: https://github.com/miguelbalparda/carpoolear_vm

There is all resolved, including the backend URL

**Comment by medrockstar** [comment]
yes, but what's the url like carpoolear/backend, to manage the application (dashboard).

**Comment by p4bl1t0** [comment]
The backend doesn't have an appplication dashboard.

**Comment by medrockstar** [comment]
is there a super admin profile ?

**Comment by p4bl1t0** [comment]
Currently there is not. The super admin stuff has to be done in a low level way (directly from database).

**Comment by medrockstar** [comment]
perfect, I install the but I can't register from http://localhost:8081/register

**Comment by medrockstar** [comment]
Ocurrió un error al procesar el registro, por favor vuelva a intentar.

**Comment by p4bl1t0** [comment]
@medrockstar please refer to this issue https://github.com/STS-Rosario/carpoolear_backend/issues/95 for the subject

**Comment by medrockstar** [comment]
the loader can't stop, I have nothing if I'm login
what this mean http://covoiturage/public/api/trips?is_passenger=false&page=1&page_size=20 500 (Internal Server Error) ?
